RecoTracker/SpecialSeedGenerators/src/CtfSpecialSeedGenerator.cc

```diff
@@ -17,6 +17,13 @@ using namespace ctfseeding;
 
 CtfSpecialSeedGenerator::CtfSpecialSeedGenerator(const edm::ParameterSet& conf)
     : conf_(conf),
```

Comment:
it would be nice to get rid of `conf_` here as well, but this spreads to `beginRun` and other methods; so, it's ok to leave for some indeterminate future

Reply:
I'll leave that for another PR.

Reply:
The `conf_` is removed (along some other cleanup) in https://github.com/cms-sw/cmssw/pull/35393.